compiler/rustc_middle/src/ty/context.rs

```diff
@@ -129,9 +129,9 @@ pub struct CtxtInterners<'tcx> {
     projs: InternedSet<'tcx, List<ProjectionKind>>,
     place_elems: InternedSet<'tcx, List<PlaceElem<'tcx>>>,
     const_: InternedSet<'tcx, Const<'tcx>>,
-    /// Const allocations.
-    allocation: InternedSet<'tcx, Allocation>,
+    const_allocation: InternedSet<'tcx, Allocation>,
```

Comment:
Nit: This asks for `const_allocations` (it looks like there may be more than 1 of them?)

Reply:
The interner fields are the names of the interned type, so plurals only show up for interned lists.